### PR TITLE
Integrate external module crate in runtime

### PR DESCRIPTION
**Pull request by bedeho**
Will update Cargo reference with proper commit hash once forum repo is finalised.

### CODE REVIEW

**Comment by siman** [comment]
Could we merge this one please if everything is fine to you guys @bedeho @mnaamani?

**Comment by mnaamani** [comment]
LGTM, just needs a `rustfmt` to pass travis

**Comment by mnaamani** [comment]
Should also add the `wasm/Cargo.lock`

File: Cargo.toml [hunk 1]
```diff
@@ -188,3 +189,8 @@ git = 'https://github.com/joystream/substrate.git'
 package = 'srml-finality-tracker'
 rev = '6dfc3e8b057bb00322136251a0f10305fbb1ad8f'
 
+[dependencies.forum]
+default_features = false
+git = 'https://github.com/joystream/substrate-forum-module'
+package = 'substrate-forum-module'
+rev = '4615fdc126ad7f3fe0fef7620177b38c307d9e2d'
```
Comment (mnaamani) [inline Cargo.toml]: This can now be updated to `5ab4944344966e0b460f3883cbc45eab94ef666e`

Comment (mnaamani) [inline Cargo.toml]:
```suggestion
rev = '5ab4944344966e0b460f3883cbc45eab94ef666e'
```

**Comment by bedeho** [comment]
What is the status here now?

**Comment by mnaamani** [comment]
> What is the status here now?

The subsystem test we ran yesterday was based on this PR + https://github.com/bedeho/substrate-runtime-joystream/pull/1

So if you want to merge that one first, we can then merge this PR, unless there is some more worked needed on the forum runtime.

**Pull request merged**